jwst/assign_wcs/util.py

```diff
@@ -659,6 +680,21 @@ def create_grism_bbox(input_model,
                     ymin = int(np.min(ystack))
                     ymax = int(np.max(ystack))
 
+                    # The ``72`` criteria is a placeholder fof when there's a decision
+                    # on how to populate the ``is_star`` column in the catalog.
+                    # See ``is_star`` property in source_catalog.py
+                    if wfss_extract_half_height is not None and obj.is_star > 72:
```

Comment:
Is there a way to test for `NaN` versus `True`, such that if `NaN`, don't apply, and if `True`, then do apply it.